bridge_ui/src/utils/alephium.ts

```diff
@@ -1,143 +1,142 @@
+    throw Error("invalid token id " + tokenId)
+  }
+  const localTokenPoolId = getTokenPoolId(ALEPHIUM_TOKEN_BRIDGE_CONTRACT_ID, CHAIN_ID_ALEPHIUM, tokenId)
+  const tokenPoolCreated = await contractExists(localTokenPoolId, nodeProvider)
+  return tokenPoolCreated ? localTokenPoolId : null
 }
 
 export class TokenInfo {
-    decimals: number
-    symbol: string
-    name: string
-
-    constructor(decimals: number, symbol: string, name: string) {
-        this.decimals = decimals
-        this.symbol = symbol
-        this.name = name
-    }
+  decimals: number
+  symbol: string
+  name: string
+
+  constructor(decimals: number, symbol: string, name: string) {
+    this.decimals = decimals
+    this.symbol = symbol
+    this.name = name
+  }
 }
 
 export async function getAlephiumTokenInfo(provider: NodeProvider, tokenId: string): Promise<TokenInfo | undefined> {
-    // TODO: get symbol and name from configs
-    if (tokenId === ALEPHIUM_WRAPPED_ALPH_CONTRACT_ID) {
-        return new TokenInfo(0, 'wrapped-alph', 'wrapped-alph')
-    }
-
-    const tokenAddress = toAlphContractAddress(tokenId)
-    try {
-        const group = await provider.addresses.getAddressesAddressGroup(tokenAddress)
-        const state = await provider.contracts.getContractsAddressState(tokenAddress, {group: group.group})
-        if (state.codeHash === ALEPHIUM_REMOTE_TOKEN_POOL_CODE_HASH) {
-            const symbol = (state.fields[4] as node.ValByteVec).value
-            const name = (state.fields[5] as node.ValByteVec).value
-            const decimals = parseInt((state.fields[6] as node.ValU256).value)
-            return new TokenInfo(decimals, symbol, name)
-        }
+  // TODO: get symbol and name from configs
+  if (tokenId === ALEPHIUM_WRAPPED_ALPH_CONTRACT_ID) {
+    return new TokenInfo(0, 'wrapped-alph', 'wrapped-alph')
+  }
 
-        const localTokenPoolId = await getLocalTokenPoolId(provider, tokenId)
-        return localTokenPoolId ? new TokenInfo(0, 'token', 'token') : undefined
-    } catch (error) {
-        console.log("failed to get alephium token info, error: " + error)
-        return undefined
+  const tokenAddress = toAlphContractAddress(tokenId)
+  try {
+    const group = await provider.addresses.getAddressesAddressGroup(tokenAddress)
+    const state = await provider.contracts.getContractsAddressState(tokenAddress, { group: group.group })
+    if (state.codeHash === ALEPHIUM_REMOTE_TOKEN_POOL_CODE_HASH) {
+      const symbol = (state.fields[4] as node.ValByteVec).value
+      const name = (state.fields[5] as node.ValByteVec).value
+      const decimals = parseInt((state.fields[6] as node.ValU256).value)
+      return new TokenInfo(decimals, symbol, name)
     }
+
+    const localTokenPoolId = await getLocalTokenPoolId(provider, tokenId)
+    return localTokenPoolId ? new TokenInfo(0, 'token', 'token') : undefined
+  } catch (error) {
+    console.log("failed to get alephium token info, error: " + error)
+    return undefined
+  }
```

Comment:
This function is for testing purpose only right?

later we will read such information from a public repo `token-list`.

Reply:
Yes, it's only for testing now, wormhole also has a repo for this: https://github.com/certusone/wormhole-networks.